services/catarse/app/models/contribution.rb

```diff
@@ -61,6 +62,10 @@ def self.need_notify_about_pending_refund
       and (current_timestamp - un.created_at) > '7 days'::interval) or not exists(select true from contribution_notifications un where un.contribution_id = contributions.id and un.template_name = 'contribution_project_unsuccessful_slip_no_account'))").distinct
   end
 
+  def updated_service_slip_fee
+    self.project.update_column(:service_slip_fee, CatarseSettings.get_without_cache(:catarse_slip_tax).to_i) if self.project.service_slip_fee == 0
+  end
+
```

Comment:
**Sugestão:**
Caso esse método permaneça: 
- Alterar nome do método para `update_project_service_slip_fee`
- Tentar se manter dentro das 120 colunas, essa linha tá muito grande, dificulta a leitura do código. Isso pode ser solucionado sem esse `if` inline, ou colocando um `return if project.ser...` no início do método.
- Esses `self` não são necessários nesses casos, pois não é atribuição de valor a um campo do modelo.